include/circt-c/Dialect/HW.h

```diff
@@ -32,6 +32,7 @@ extern "C" {
 struct HWStructFieldInfo {
   MlirStringRef name;
   MlirAttribute attribute;
+  MlirType type;
```

Comment:
See my question in https://github.com/llvm/circt/commit/2343f692d0adcc436f56760991c1e763e3772479